_sass/blocks/_footer.scss

```diff
@@ -6,38 +6,142 @@
 // Styleguide blocks.footer
 
 footer,
+.footer {
+  background-color: $light-black;
+  padding-top: $base-padding-extra;
+  padding-bottom: $base-padding-extra;
+
+  .fa-circle {
+    color: $neutral-gray;
+  }
+
+  .fa-inverse {
+    color: $light-black;
+  }
+}
+
+footer,
 .footer,
 .footer-heading,
-.footer-para {
+.footer-para,
+.footer-para_callout,
+.footer-social {
   color: $white;
 }
 
-footer,
-.footer {
-  background-color: $light-black;
-  padding-bottom: $base-padding;
+.footer-para {
+  margin-bottom: $base-padding;
+}
+
+.footer-para_last {
+  @include respond-to(medium-up) {
+    margin-bottom: 0;
+  }
+}
+
+.footer-para_callout {
+  font-weight: $weight-book;
+  margin-bottom: 0;
+}
+
+.footer-image {
+  margin-bottom: $base-padding-large;
+  max-width: 300px;
+  padding-right: $base-padding;
+  width: 100%;
+
+  @include respond-to(medium-up) {
+    max-width: none;
+  }
+}
+
+.footer-button {
+  height: auto;
+  padding-left: 15px;
+  padding-right: 15px;
+  width: auto;
 }
 
-.footer-col {
-  @include span-columns(2);
-  margin-top: $base-padding;
+.footer-col-left {
+  border-bottom: 1px solid $white;
+  margin-bottom: $base-padding-extra;
 
-  &:first-of-type {
-    @include span-columns(2);
+  @include respond-to(medium-up) {
+    border-bottom: none;
   }
 }
 
-.footer-col-right {
-  @include span-columns(4);
-  @include omega();
+.footer-table {
+  border: none;
+  border-bottom: 1px solid $neutral-gray;
+  display: table;
+  margin-bottom: $base-padding-extra;
+  padding-bottom: $base-padding * 2;
+}
+
+.footer-table_head {
+  background-color: transparent;
+  border: none;
+  display: table-header-group;
+  text-align: left;
+}
+
+.footer-heading,
+.footer-list_item {
+  display: table-cell;
+}
+
+.footer-list_body {
+  display: table-row-group;
+}
+
+.footer-list_row,
+.footer-list_bottom_row {
+  display: table-row;
 }
 
 .footer-heading {
   padding-bottom: $base-padding-lite;
 }
 
-.footer-list_item {
+.footer-list_item,
+.footer-list_item_heading,
+.footer-list_item_empty {
+  @include heading('para-md');
+  border: 0;
```

Comment:
Properties should be ordered background-color, border, display, line-height, padding-bottom, padding-right, width